cli/commands/assessment/assessment.go

```diff
@@ -26,50 +26,23 @@
 package assessment
 
 import (
-	"context"
-	"fmt"
-
-	"clouditor.io/clouditor/api/assessment"
-	"clouditor.io/clouditor/cli"
+	"clouditor.io/clouditor/cli/commands/service/orchestrator"
 	"github.com/spf13/cobra"
 )
 
 // NewListResultsCommand returns a cobra command for the `list` subcommand
 func NewListResultsCommand() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "list",
-		Short: "Lists all assessment results",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			var (
-				err     error
-				session *cli.Session
-				client  assessment.AssessmentClient
-				res     *assessment.ListAssessmentResultsResponse
-			)
-
-			if session, err = cli.ContinueSession(); err != nil {
-				fmt.Printf("Error while retrieving the session. Please re-authenticate.\n")
-				return nil
-			}
-
-			client = assessment.NewAssessmentClient(session)
-
-			res, err = client.ListAssessmentResults(context.Background(), &assessment.ListAssessmentResultsRequest{})
-
-			return session.HandleResponse(res, err)
-		},
-		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
-			return []string{}, cobra.ShellCompDirectiveNoFileComp
-		},
-	}
-
+	// Use Orchestrator's method for listing assessment results
+	cmd := orchestrator.NewListAssessmentResultsCommand()
+	// Change use for better readability (cl assessment_result list vs cl assessment_result list_assessment_results)
+	cmd.Use = "list"
 	return cmd
 }
 
 // NewAssessmentCommand returns a cobra command for `assessment` subcommands
 func NewAssessmentCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "assessment",
+		Use:   "assessment_result",
```

Comment:
I think in the commands we should stick to dashes instead of underscores for a little bit of better readability.